### ISSUE TITLE
弱智吧 2020 上半年精选

### ISSUE THREAD

**Issue by yd0sJ9NqtaWO1kCPi8i67** [cirosantilli/china-dictatorship#140]
![1](https://user-images.githubusercontent.com/68586939/88245528-1a9a4780-ccca-11ea-9575-30d7ff9bf751.jpg)
![2](https://user-images.githubusercontent.com/68586939/88245530-1e2dce80-ccca-11ea-91c4-aa193070475c.jpg)
![3](https://user-images.githubusercontent.com/68586939/88245536-2259ec00-ccca-11ea-8aa6-12932351124d.jpg)
![4](https://user-images.githubusercontent.com/68586939/88245541-2554dc80-ccca-11ea-993e-493c274940e1.jpg)
![5](https://user-images.githubusercontent.com/68586939/88245542-271ea000-ccca-11ea-9b08-06bd0c2e89e6.jpg)
![6](https://user-images.githubusercontent.com/68586939/88245546-28e86380-ccca-11ea-940b-44ed28cad1e2.jpg)
![7](https://user-images.githubusercontent.com/68586939/88245547-2a199080-ccca-11ea-8747-f62538c5f695.jpg)
![8](https://user-images.githubusercontent.com/68586939/88245553-2c7bea80-ccca-11ea-9aca-e5eddb210a41.jpg)
![9](https://user-images.githubusercontent.com/68586939/88245558-2f76db00-ccca-11ea-8013-f739750f0b0d.jpg)

**Comment by cirosantilli** [comment]
Hi yd0sJ9NqtaWO1kCPi8i67

Thanks for this, can you summarize briefly what is the meaning of this list?

Is it a random collection of "stupid things said on the internet"? Or does it have a more precise theme? Were previous spam posts were taking their titles from?
